extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/ApacheHttpClientConfig.java

```diff
@@ -0,0 +1,149 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ApacheHttpClientConfig {
+
+    /**
+     * The amount of time to wait when acquiring a connection from the pool before giving up and timing out.
+     *
+     * Default is 10 seconds.
+     */
+    @ConfigItem
+    public Optional<Duration> connectionAcquisitionTimeout;
+
+    /**
+     * Configure the maximum amount of time that a connection should be allowed to remain open while idle.
+     *
```

Comment:
In straight Javadoc, you would need a `<p>`, wouldn't you? Or the doc output is OK?

Reply:
Yes I missed that. Fixed.

Reply:
I mean added paragraph to the next row. Basically the doc output when checking the generated HTML is fine